torch/_inductor/pattern_matcher.py

```diff
@@ -1353,11 +1353,28 @@ def run_node(self, old_node):
 def get_arg_value(
     node: torch.fx.Node, arg_number: int, kwarg_name: Optional[str] = None
 ):
-    return (
-        node.args[arg_number]
-        if len(node.args) > arg_number
-        else node.kwargs.get(kwarg_name)
-    )
+    # see python_arg_parser.cpp; e.g. kwargs named "dim" also match "axis"
+    numpy_compat = {
```

Comment:
Can this just be a pass on pre-grad stack ? these args get canonicalized in torch dispatch

Reply:
@eellison is there a way for me to set the priority of the pass so that we can guarantee that the numpy-arg-canonicalization pass will run before the split-cat pass?

Reply:
You can follow the current ```normalization_pass```:
https://github.com/pytorch/pytorch/blob/main/torch/_inductor/fx_passes/split_cat.py#L165

As @eellison mentioned, only pre_grad fx passes suffer from this issue, we just need to handle them.